.vscode/extensions.json

```diff
@@ -0,0 +1,9 @@
+{
+    "recommendations": [
+        "editorconfig.editorconfig",
+        "hashicorp.terraform",
+        "ms-python.black-formatter",
+        "matangover.mypy",
```

Comment:
Is this needed if pylance is already recommended?

Reply:
No I don't think so. I'll remove it.